install/update_91_92.php

```diff
@@ -678,160 +678,199 @@ function update91to92() {
-                 `uuid` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
-                 PRIMARY KEY (`id`),
-                 KEY `name` (`name`),
-                 KEY `is_active` (`is_active`),
-                 KEY `olas_id` (`olas_id`)
-               ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;";
-         $DB->queryOrDie($query, "9.2 add table glpi_olalevels");
-      }
+   if (!$DB->tableExists('glpi_olalevels')) {
+      $query = "CREATE TABLE `glpi_olalevels` (
+               `id` int(11) NOT NULL AUTO_INCREMENT,
+               `name` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
+               `olas_id` int(11) NOT NULL DEFAULT '0',
+               `execution_time` int(11) NOT NULL,
+               `is_active` tinyint(1) NOT NULL DEFAULT '1',
+               `entities_id` int(11) NOT NULL DEFAULT '0',
+               `is_recursive` tinyint(1) NOT NULL DEFAULT '0',
+               `match` char(10) COLLATE utf8_unicode_ci DEFAULT NULL COMMENT 'see define.php *_MATCHING constant',
+               `uuid` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
+               PRIMARY KEY (`id`),
+               KEY `name` (`name`),
+               KEY `is_active` (`is_active`),
+               KEY `olas_id` (`olas_id`)
+            ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;";
+      $DB->queryOrDie($query, "9.2 add table glpi_olalevels");
+   }
 
-      if (!$DB->tableExists('glpi_olalevels_tickets')) {
-         $query = "CREATE TABLE `glpi_olalevels_tickets` (
-                    `id` int(11) NOT NULL AUTO_INCREMENT,
-                    `tickets_id` int(11) NOT NULL DEFAULT '0',
-                    `olalevels_id` int(11) NOT NULL DEFAULT '0',
-                    `date` datetime DEFAULT NULL,
-                    PRIMARY KEY (`id`),
-                    KEY `tickets_id` (`tickets_id`),
-                    KEY `olalevels_id` (`olalevels_id`),
-                    KEY `unicity` (`tickets_id`,`olalevels_id`)
-                  ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;";
-         $DB->queryOrDie($query, "9.2 add table glpi_olalevels_tickets");
-      }
+   if (!$DB->tableExists('glpi_olalevels_tickets')) {
+      $query = "CREATE TABLE `glpi_olalevels_tickets` (
+                  `id` int(11) NOT NULL AUTO_INCREMENT,
+                  `tickets_id` int(11) NOT NULL DEFAULT '0',
+                  `olalevels_id` int(11) NOT NULL DEFAULT '0',
+                  `date` datetime DEFAULT NULL,
+                  PRIMARY KEY (`id`),
+                  KEY `tickets_id` (`tickets_id`),
+                  KEY `olalevels_id` (`olalevels_id`),
+                  KEY `unicity` (`tickets_id`,`olalevels_id`)
+               ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;";
+      $DB->queryOrDie($query, "9.2 add table glpi_olalevels_tickets");
 
       $query = "INSERT INTO `glpi_crontasks`
```

Comment:
here, maybe ``REPLACE``

Reply:
I do not think this is required since the add would only be run when table was not present; this should never be a duplicate.
Anyways; I can change that, this would have no impact